docs/README.Linux.md

```diff
@@ -223,6 +232,47 @@ sudo make -j$(getconf _NPROCESSORS_ONLN) -C tools/depends/target/binary-addons P
 
 **[back to top](#table-of-contents)**
 
+
+### 5.2. Out-of-tree building of binary add-ons
+
+First build the the platform add-on, required to be able to build and install other add-ons:
```

Comment:
It's not required for all add-ons.
Also in In-tree building https://github.com/xbmc/platform is used.